pkgs/development/tools/electron/common.nix

```diff
@@ -272,21 +279,6 @@ in
 
   passthru = {
     inherit info fetchedDeps;
-    headers = stdenv.mkDerivation rec {
-      name = "node-v${info.node}-headers.tar.gz";
-      nativeBuildInputs = [ python3 ];
-      src = fetchedDeps."src/third_party/electron_node";
-      buildPhase = ''
-        runHook preBuild
-        make tar-headers
-        runHook postBuild
-      '';
-      installPhase = ''
-        runHook preInstall
-        mv ${name} $out
-        runHook postInstall
-      '';
-    };
   };
 
   meta = with lib; {
```

Comment:
remove with lib

Reply:
No, thanks. Copying and pasting that instruction around doesn't make it more correct.